beetsplug/smartplaylist.py

```diff
@@ -186,7 +187,7 @@ def update_playlists(self, lib):
             # As we allow tags in the m3u names, we'll need to iterate through
             # the items and generate the correct m3u file names.
             for item in items:
-                m3u_name = item.evaluate_template(name, True)
+                m3u_name = item.evaluate_template(sanitized_name, True)
```

Comment:
Hmm; are you sure you want to be sanitizing the template instead of the result of filling in the template?